### DISCUSSION TITLE
Query for import and export a spreadsheet

### DISCUSSION THREAD

**Discussion by Mahejoa** [frePPLe/frepple#546] [Q&A]
Need to do import and export a spreadsheet using REST API, Need a clarification please update the same.

**Comment by jdetaeye** [comment]
The REST API doesn't have spreadsheet interface.  It follows the classic REST principles: https://en.wikipedia.org/wiki/Representational_state_transfer.

Frepple does have an API interface for importing excel or csv data files: see https://frepple.org/docs/current/command-reference.html#importfromfolder
And similarly for exporting excel or csv data files: https://frepple.org/docs/6.23/command-reference.html#exporttofolder

**Reply by Mahejoa** [reply]
Thanks for the Reply. We are facing some issues in mention the spreadsheet path for import. Is there any extensions or references. Please update the same.
